packages/website/styles/global.css

```diff
@@ -1,12 +1,114 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+/* ================================================================================
+   Fonts
+   ================================================================================ */
+
+@font-face {
+    font-family: "Space Grotesk";
+    font-weight: 400;
+    font-style: normal;
+    src: url("/fonts/space-grotesk/space-grotesk-regular.woff2") format("woff2");
+    src: url("/fonts/space-grotesk/space-grotesk-regular.woff") format("woff");
+}
+
+@font-face {
+    font-family: "Space Grotesk";
+    font-weight: 500;
+    font-style: normal;
+    src: url("/fonts/space-grotesk/space-grotesk-medium.woff2") format("woff2");
+    src: url("/fonts/space-grotesk/space-grotesk-medium.woff") format("woff");
+}
+
+@font-face {
+    font-family: "Helvetica";
+    font-weight: 400;
+    font-style: normal;
+    src: url("/fonts/helvetica/helvetica.woff2") format("woff2");
```

Comment:
I would change this approach from using self-hosted font files to just using google fonts.

Pros:
- Easier to cache through websites (let's say the user already browsed a website with Helvetica in google fonts, they would get that cached in the browser and it speeds up the first load ever)
- Removes font bloatware, which you currently have with 8 fonts.

Cons:
- The only con is that it adds an extra domain to fetch from the initial load, but with current gen techonologies, adding pre-load links to google fonts easily fixes it.

Reply:
The website should be self contained, so we can add it to IPFS and host it entirely from the root cid. In general we want to limit our dependence on third party domains for all our sites.

Reply:
AKA: please let's self host the fonts, unless there is a very strong reason not to.

Reply:
That's a strong point @olizilla, go fo self-host then 👍